src/sinol_make/commands/run/__init__.py

```diff
@@ -38,6 +38,8 @@ def configure_subparser(self, subparser):
 							help='hide memory usage in report')
 		parser.add_argument('--program_report', type=str,
 							help='file to store report from program executions (in markdown)')
+		parser.add_argument('--time_tool', choices=['oiejq', 'time'], default='oiejq',
+		      				help='tool to measure time and memory usage (default: oiejq)')
```

Comment:
Is it also default for macOS, for which oiejq doesn't work?

Reply:
Yes, I will change it

Reply:
Changed 3be1569831372fb44a4161a43df1331e561cd0de